### PR TITLE
Try fixing the crashes on VTK-9.1.0.

### PR DESCRIPTION
**Pull request by prabhuramachandran**
- The crashes seem to be related to the `vtkOpenGLAvatar` class being instantiated.  
- Do not instantiate all the VTK objects on import which the tvtk_doc module was doing unnecessarily.

### CODE REVIEW

**Comment by prabhuramachandran** [comment]
Yay! This works, we should push out a release soon.

File: tvtk/vtk_module.py [hunk 1]
```diff
@@ -31,3 +31,10 @@
     # Cause problems if used so ignore these.
     SKIP = ['vtkDataEncoder', 'vtkWebApplication']
     del vtkDataEncoder, vtkWebApplication
+
+if vtk_version == '9.1.0':
+    SKIP = ['vtkOpenGLAvatar']
+    try:
+        del vtkOpenGLAvatar
+    except NameError:
```
Comment (rahulporuri) [inline tvtk/vtk_module.py:39]: i'm not sure why we're expecting an exception to be raised here.

**Review by rahulporuri — APPROVED** [review]
LGTM with one question

File: tvtk/vtk_module.py (hunk 1 above)
Comment (prabhuramachandran) [reply tvtk/vtk_module.py:39]: Well, I am not sure if there are build options that may exclude the building of these classes so I was just being extra safe.  The issue may arise if someone has a hand-built VTK.

**Pull request merged**